include/boost/process/detail/posix/is_running.hpp

```diff
@@ -70,7 +70,22 @@ inline bool is_running(int code)
 
 inline int eval_exit_status(int code)
 {
-    return WEXITSTATUS(code);
+    if (WIFEXITED(code))
+    {
+        return WEXITSTATUS(code);
+    }
+    else if (WIFSIGNALED(code))
+    {
+        return WTERMSIG(code);
+    }
+    else if (WIFSTOPPED(code))
```

Comment:
If I correctly understand `WIFSTOPPED`, it's not _quite_ an exit code since the process is still "alive" (just suspended). But :+1: I think this is definitely better than it was before.

Reply:
It only extracts the actual value, thus that is completely fine.